libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -190,21 +237,6 @@ func (m *Manager) Apply(pid int) error {
 			newProp("DefaultDependencies", false))
 	}
 
-	if c.Resources.Memory != 0 {
-		properties = append(properties,
-			newProp("MemoryLimit", uint64(c.Resources.Memory)))
-	}
-
-	if c.Resources.CpuShares != 0 {
-		properties = append(properties,
-			newProp("CPUShares", uint64(c.Resources.CpuShares)))
-	}
-
-	if c.Resources.BlkioWeight != 0 {
-		properties = append(properties,
-			newProp("BlockIOWeight", uint64(c.Resources.BlkioWeight)))
-	}
-
```

Comment:
I'm not sure if this is a good idea, it might cause systemd to change the paths the container joins (which might cause further issues).